test/crc_test2.cpp

```diff
@@ -54,7 +48,7 @@ boost::uint16_t const  std_crc_16_result = 0xBB3Du;
 boost::uint32_t const  std_crc_32_result = 0xCBF43926ul;
 
 // Conversion functions between native- and big-endian representations
-#ifdef BOOST_BIG_ENDIAN
+#ifdef BOOST_BIG_ENDIAN_BYTE
```

Comment:
#if here. BOOST_*_ENDIAN_BYTE are always defined.

Reply:
Also, it's BOOST_ENDIAN_BIG_BYTE. :-)

Reply:
Ah. :)